crypto/rsa/rsa.c

```diff
@@ -121,23 +121,17 @@ int rsa_new_end(RSA *rsa, BN_CTX *ctx) {
    * BN_FLG_CONSTTIME flag. */
```

Comment:
`assert(BN_get_flags(rsa->p, BN_FLG_CONSTTIME));` etc. for all BIGNUM variables in the RSA structure.